documentation/source/class.rst

```diff
@@ -29,6 +29,14 @@ additionally the ``members``, ``protected-members``, ``private-members`` and
 ``undoc-members``
    If specified, the undocumented members of the class will be displayed.
 
+``membergroups``
+  If specified, only the groups in a comma-delimired list following this
+  directive will be displayed.
+  
+``members-only``
+  This will allow to show only the members. It will remove group names,
+  descriptions as well as the inheritance information.
```

Comment:
Could you also update the paragraph at the top of this file, adding the new options?

Reply:
will do

Reply:
will do